api/src/Entity/User.php

```diff
@@ -643,7 +653,7 @@ public function getUsername()
 
     public function getSalt()
     {
-        //return $this->salt;
+        // return $this->salt;
```

Comment:
Why is this commented out?

Reply:
No idea - the change was made in 2015 and hasn't been touched since.

Reply:
That's an odd one then. Worth spinning up a ticket to investigate.